app/views/vet-visits.njk

```diff
@@ -47,15 +59,17 @@
         <hr/>
       {% endif %}
       <h2 class="govuk-heading-m govuk-!-static-margin-top-4">More information</h2>
-      <p class="govuk-body">
-        <a class="govuk-link" rel="external" href={{ latestTermsAndConditionsUri }}>Terms and conditions</a>
-      </p>
-      <p class="govuk-body">
-        <a  class="govuk-link" rel="external" href="https://www.gov.uk/government/collections/funding-to-improve-animal-health-and-welfare-guidance-for-farmers-and-vets">Guidance for funding to improve animal health and welfare</a>
-      </p>
-      <p class="govuk-body">
-        <a class="govuk-link" rel="external" target=”_blank”  href={{ downloadedDocument }}>Agreement summary</a>
-      </p>
+      <ul class="govuk-list">
```

Comment:
Was changing the layout of these links supposed to be part of this ticket? I prefer the layout produced with these changes, but are UCD happy that this has changed away from having blank lines between the links as we had previously?

Reply:
Saw you demo to them and they seemed delighted anyway, so all good